### ISSUE TITLE
Question: Why is ResponseDate not available in the skill?

### ISSUE THREAD

**Issue by danielbrunt57** [keatontaylor/alexa-actions#108]
My question: Why is ResponseDate not available in the skill?
I tried adding it to the lambda_function.py...
```
RESPONSE_YES = "ResponseYes"
RESPONSE_NO = "ResponseNo"
RESPONSE_NONE = "ResponseNone"
RESPONSE_SELECT = "ResponseSelect"
RESPONSE_NUMERIC = "ResponseNumeric"
RESPONSE_DURATION = "ResponseDuration"
RESPONSE_DATE = "ResponseDate"
```
but when I respond with a date/time like **_Friday at 9:30 am_** or **_At 7 am_** Alexa still answers with **_Sorry, I cannot do specific dates right now. Try a duration instead, like, in 5 hours_**. 

@keatontaylor 
Regarding the comment in the this default issue template: `For all support requests it is strongly recommend you join the #alexa-actionable-notifications discord channel for assistance. Myself and others are able to help you in real-time with issues.  [Discord Invite Link](https://discord.gg/AWgE6g)`, that Invite link is dead and I cannot find that channel on Discord...

**Comment by DEADSEC-SECURITY** [comment]
You need to develop a handler for the ResponseDate, if you need any help developing it, its actually a good future implementation to do. Maybe Ill add it to the source code in the future.

**Comment by DEADSEC-SECURITY** [comment]
Could you confirm this is still an issue?

**Comment by danielbrunt57** [comment]
Yes, it is still an issue/desirable.  I spent a lot of time today scouring the 'net for clues on what a a handler might look like but got nowhere...

**Comment by DEADSEC-SECURITY** [comment]
Are you using a show device @danielbrunt57

**Comment by danielbrunt57** [comment]
No I am not.

**Comment by danielbrunt57** [comment]
Alexa still says this: **_Sorry, I cannot do specific dates right now. Try a duration instead, like, in 5 hours._**

**Comment by DEADSEC-SECURITY** [comment]
@danielbrunt57 this is now implemented in the PR #131
